src/components/FilterCheckboxes.js

```diff
@@ -0,0 +1,83 @@
+import React, {Component} from 'react';
+import { FormControlLabel, FormGroup, FormControl, Checkbox, Button, Collapse } from '@material-ui/core';
+import CheckBoxIntermediateIcon from 'mdi-react/CheckboxIntermediateIcon';
+import { withStyles } from '@material-ui/core/styles';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import PlaceIcon from '@material-ui/icons/Place';
+
+const styles = {
+    checkbox: {
+        margin: '0px 0px 0px 8px',
+    },
+    checkedCheckbox: {
+        color: '#93C838'
+    },
+    showAllButton: {
+        margin: 'auto'
+    },
+    allContent: {
+        width: '100%'
+    }
+}
+
+class FilterCheckboxes extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            viewAll: false
+        }
+    }
+
+    getButton = (itemKey, checked, disabled, onChange, classes, keyColorFunction) => {
+        const label = keyColorFunction ? <span><PlaceIcon style={{color: keyColorFunction(itemKey)}}/>{itemKey}</span> : <span>{itemKey}</span>;
+        return <FormControlLabel key={itemKey}
+                    control={<Checkbox
+                        checkedIcon={<CheckBoxIntermediateIcon className={classes.checkedCheckbox}/>}
+                        checked={checked}
+                        onChange={onChange}
+                        disabled={disabled}
+                        className={classes.checkbox}/>} label={label}/>
+    }
+
+    render() {
+        const {allLabel, briefNumber, filter, updateValues, classes, keyColorFunction} = this.props;
+        const {viewAll} = this.state;
+        return <FormControl component="fieldset" className={classes.allContent}>
+            <FormGroup>
+                <FormControlLabel
```

Comment:
We have complete control over what gets checked when. I wasn't sure about the best behavior so my first instinct was that checking "all" should disable the other options, since it's not immediately clear what [all, black bears] should display. At the same time, my thought was that it would be a little annoying if you've set up a filter and you click "all" and all of your choices were cleared. But if you think it's better another way let me know and I can rework it

Reply:
I think it should be fine as is now.